tests/functional/monitoring/prometheus/alerts/test_alert_mds_cache_high_usage.py

```diff
@@ -0,0 +1,234 @@
+import time
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.utility import prometheus
+from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.utility.utils import ceph_health_check_base
+from ocs_ci.ocs.node import (
+    unschedule_nodes,
+    drain_nodes,
+    schedule_nodes,
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def run_metadata_io_with_cephfs(pvc_factory, dc_pod_factory):
+    """
+    This function facilitates
+    1. Create PVC with Cephfs, access mode RWX
+    2. Create dc pod with Fedora image
+    3. Copy helper_scripts/meta_data_io.py to Fedora dc pod
+    4. Run meta_data_io.py on fedora pod
+    """
+    access_mode = constants.ACCESS_MODE_RWX
+    file = constants.METAIO
+    interface = constants.CEPHFILESYSTEM
+    log.info("Checking for Ceph Health OK")
+    ceph_health_check_base()
```

Comment:
this check is already done at the beginning of the test execution by our framework, I think we can remove it here